driver.sh

```diff
@@ -78,7 +86,7 @@ check_dependencies() {
 }
 
 mako_reactor() {
-  make -j"$(nproc)" CC="$ccache $clang" HOSTCC="$ccache $clang" "$@"
+  make -j"$(nproc)" CC="${ccache} ${clang}" HOSTCC="${ccache} ${clang}" "$@"
```

Comment:
Just curious if the `$@` should be brace expanded as well? Ie `"${@}"`?

Reply:
Done.